modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cache/ClientCacheScanQueryRequest.java

```diff
@@ -74,7 +74,8 @@ public ClientCacheScanQueryRequest(BinaryRawReaderEx reader) {
 
     /** {@inheritDoc} */
     @Override public ClientResponse process(ClientConnectionContext ctx) {
-        IgniteCache cache = filterPlatform == ClientPlatform.JAVA && !isKeepBinary() ? rawCache(ctx) : cache(ctx);
+        IgniteCache<Object, Object> cache = filterPlatform == ClientPlatform.JAVA && !isKeepBinary() ?
+            rawCache(ctx) : cache(ctx);
```

Comment:
```suggestion
        IgniteCache<Object, Object> cache = filterPlatform == ClientPlatform.JAVA && !isKeepBinary() 
            ? rawCache(ctx) 
            : cache(ctx);
```

Reply:
It's not specified by our code style and is a matter of taste. For Ignite is more common to use `?` at the end of the line. For example, I found 375 occurances of `\?$` pattern in .java files of ignite-core module versus 213 occurances of `^\s+\?`.